src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/VariantAnnotatorEngine.java

```diff
@@ -101,7 +139,7 @@ public static VariantAnnotatorEngine ofSelectedMinusExcluded(final List<String>
      *                   Note: there are no non-DBSNP comparison FeatureInputs an empty List should be passed in here, rather than null.
      * @return a VariantAnnotatorEngine initialized with the requested annotations
      */
-    public static VariantAnnotatorEngine ofSelectedMinusExcluded(final VariantAnnotationArgumentCollection argumentCollection,
+    public static VariantAnnotatorEngine ofSelectedMinusExcluded(final DefaultGATKVariantAnnotationArgumentCollection argumentCollection,
```

Comment:
Some comment on types here. The type of this arg should be GATKVariantAnnotationArgumentCollection.

Reply:
done